client/wfm/src/process-instance/ProcessInstanceRepository.ts

```diff
@@ -0,0 +1,8 @@
+import {ProcessInstance} from './ProcessInstance';
+export interface ProcessInstanceRepository {
+  getAll(): Promise<ProcessInstance[]>;
+  getById(id: string): Promise<ProcessInstance>;
+  getByProcessId(id: string): Promise<ProcessInstance[]>;
```

Comment:
IS that used? IMHO we may never use this actual method as it will be overkill.
We always looking for filtered ProcessInstances (probably per assignee etc.)